### DISCUSSION TITLE
Paddle Sandbox returns HTTP 400 on product booking

### DISCUSSION THREAD

**Discussion by phenomenia** [thedevdojo/wave#119] [Q&A] [answered]
Hi,
i just found Wave and directly bought it. I've set up the Paddle Sandbox in Wave according to the docs, while waiting for the verification of the real account. I've also setup the product as padle and linked them with the ID (starting with pri_) to the products in Wave.
I've also added the PADDLE_CLIENT_SIDE_TOKEN in the .env file (which is missing in the video doc). This made that when I click a product to do a test buy the paddle layer opened at all. However, I still get an error:
"Something went wrong. Please try again later. If the issue persists, you can contact our support team using the button below."
And in the browser console I see an Error 400 for https://sandbox-checkout-service.paddle.com/transaction-checkout

Any ideas how to fix this?
Have I overlooked something to setup?
![CleanShot 2024-05-19 at 17 50 43@2x](https://github.com/thedevdojo/wave/assets/736951/f957bcb6-4a15-465c-9b7a-39271eb0c00f)

**Comment by bobbyiliev** [comment, marked as answer]
Hi there,

Indeed, the videos might be a bit outdated around the Paddle configuration as Paddle did a huge rework on their end recently. And those changes are reflected only in the Wave docs but on in the videos.

The Wave docs are updated and you can follow them here:

> https://wave.devdojo.com/docs/features/billing

Note that you need to have the Paddle billing rather than the Paddle Classic. It is a bit confusing on the Paddle side how they handled this implementation. Basically, if you have a new account it will be the Paddle billing one.

Also are you using the Tailwind Wave theme or the Tallstack theme?
